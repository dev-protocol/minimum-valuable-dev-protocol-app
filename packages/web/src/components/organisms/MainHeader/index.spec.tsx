import React from 'react'
import { render } from '@testing-library/react'
import { MainHeader } from '.'

jest.mock('src/fixtures/dev-kit/hooks')

describe(`${MainHeader.name}`, () => {
  test('Snapshot', () => {
    const component = render(<MainHeader />)
    const tree = component.baseElement
    expect(tree).toMatchSnapshot()
  })
})
