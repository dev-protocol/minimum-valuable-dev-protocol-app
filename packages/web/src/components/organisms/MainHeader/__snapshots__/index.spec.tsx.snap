// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainHeader Snapshot 1`] = `
<body>
  <div>
    <div
      style="padding: 50px;"
    >
      <div>
        <div
          style="display: inline; color: rgb(0, 0, 0); font-size: 48px; line-height: 64px;"
        >
          App Name
        </div>
        <button
          class="ant-btn ant-btn-lg"
          style="float: right;"
          type="button"
        >
          <span>
            Connect to a wallet
          </span>
        </button>
      </div>
      <div
        style="height: 500px; display: flex; justify-content: center; align-items: center; font-size: 48px; line-height: 64px;"
      >
        <p
          style="color: rgb(47, 128, 237);"
        >
          Make a community sustainable together
        </p>
      </div>
    </div>
  </div>
</body>
`;
